src/examples/dump.c

```diff
@@ -47,10 +47,15 @@ print_usage(char *program_name)
             basename(program_name));
 }
 
-static ssize_t
-stdin_reader(void *app_ctx, void *buf, size_t len)
+static bool
+stdin_reader(void *app_ctx, void *buf, size_t len, size_t *readres)
 {
-    return read(STDIN_FILENO, buf, len);
+    int64_t res = read(STDIN_FILENO, buf, len);
```

Comment:
I think we should use `ssize_t` here since that is what `read` returns according to POSIX.

Reply:
@dewyatt I was also thinking about it. But what will happen on system without ssize_t ?